pep-0693.rst

```diff
@@ -43,10 +43,10 @@ in a 12-month release cadence between major versions, as defined by
 Actual:
 
 - 3.12 development begins: Monday, 2022-05-08
+- 3.12.0 alpha 1: Tuesday, 2022-10-25
```

Comment:
```suggestion
- 3.12.0 alpha 1: Tuesday, 2022-10-24
```

Reply:
Timezones are hard for the website, I guess :) It was cut on the 24th, and announcements went out the 24th, local time, so let's say 24th :)